test/integration/integration_test.go

```diff
@@ -234,6 +315,8 @@ func selectFieldFromLastOutput(commandField string) string {
 		outputField = lastCommandOutput.StdErr
 	case "exitcode":
 		outputField = strconv.Itoa(lastCommandOutput.ExitCode)
+	case "proxy log":
```

Comment:
I do not believe we duplicate code, as when you write the step:

```
"^proxy log should contain$"
"^([^"]*) should contain$"
```

You can have the function just do:
```
func proxyLogShouldContain(expected string) error {
  return compareExpectedWithActualContains(expected, proxyLog.String())
}
```